public/templates/settings/settings.html

```diff
@@ -15,7 +15,7 @@
     <div layout="column" layout-align="start stretch">
 
         <!-- First-time welcome card -->
-        <md-card class="wz-md-card" layout-padding layout-align="start stretch" ng-if="submenuNavItem == 'api' && apiEntries.length == 0">
+        <md-card class="wz-md-card" layout-padding layout-align="start stretch" ng-if="submenuNavItem == 'api' && apiEntriesLoaded && apiEntries.length == 0">
```

Comment:
@JuanjiJG  we could use `load` variable who is already defined, replace `&& apiEntriesLoaded` by `&& !load`.
Since we can use it, you don't need any change con `settings.js` file, so fix it please. You should try it before any change because I didn't try it, but it should work, thanks.

Reply:
You're right, colleague. Now it's fixed, and the new variable was removed from the code. Now there are only CLRF fixes on the `settings.js` file. Thanks for your valuable feedback, it's really appreciated.

Regards,
Juanjo